account_statement_import_online/wizards/online_bank_statement_pull_wizard.py

```diff
@@ -12,15 +13,13 @@ class OnlineBankStatementPullWizard(models.TransientModel):
     _name = "online.bank.statement.pull.wizard"
     _description = "Online Bank Statement Pull Wizard"
 
-    date_since = fields.Datetime(
-        string="From",
-        required=True,
-        default=fields.Datetime.now,
-    )
-    date_until = fields.Datetime(
-        string="To",
-        required=True,
-        default=fields.Datetime.now,
+    date_since = fields.Date(string="From", required=True, default=fields.Date.today)
```

Comment:
I would very much prefer having date_since as date but have datetime_since hidden unless debug is active, quite useful when dealing with consequences of some outdages

Reply:
Indeed, I take it back :)